text/0000-landing-page.md

```diff
@@ -0,0 +1,53 @@
+# Meta
+[meta]: #meta
+- Name: Create a community landing page
+- Start Date: 1-6-2020
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+People interested in the project don't have a central place to go for information. This RFC proposes the creation of a community or "landing page" repo at `buildpacks/buildpacks` with high level information about the project including key personel, governance, guidelines for contributions, meeting schedules, etc.
+
+**Outcomes**
+[outcomes]: #outcomes
+
+ - New contributors feel welcomed.
+ - A new contributor or user of the project can easily find what they want, join the slack, WG meetings, and immediately contribute pull-requests, RFCs, and issues.
+
+# Motivation
+[motivation]: #motivation
+
+A major part of the success of the CNB project will necessitate growing contributors beyond the current core. As new people learn about the project and want to use it or contribute, we need a friendly central landing place where all the information _about_ the project is housed.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This RFC proposes the creation of a project landing page that will be house at `buildpacks/buildpacks`.
+It will have a central `readme.md` that included the following information and links to documents that provide more details if and when that's necessary
+
+- Purpose of the project and high level overview including some canonical talk from youtube, etc
+- Technical overview, including guides to the various components of the projects and links to their respective repos
+- Project and community overview, including key personel, governance structures, link to slack, calendar links, and links to a contribution guide (this document is likely sufficiently detailed as to necessitate a seperate document beloew)
```

Comment:
Many CNCF projects put this information in `<org>/community`, for example:

- https://github.com/goharbor/community
- https://github.com/open-telemetry/community

However, I think I prefer to have this in `buildpacks/buildpacks`.

Reply:
I think I prefer `/community`. If I go to a repo called `buildpacks` I feel like I'd expect to find some sort of product there.

Reply:
As much as I love my pet idea here, it does seem like a true convention to have `/community`, especially for CNCF projects, and would solve the discoverability problem @jkutner [noted below](https://github.com/buildpacks/rfcs/pull/42#discussion_r364506841).

Reply:
I updated this RFC to suggest `buildpacks/community` as the site for this, as it seems much more conventional and discoverable (if not as cool 😎 ).